library/src/main/java/io/constructor/core/ConstructorIo.kt

```diff
@@ -138,7 +138,7 @@ object ConstructorIo {
             Constants.QueryConstants.AUTOCOMPLETE_SECTION to sectionName,
             Constants.QueryConstants.ORIGINAL_QUERY to originalQuery,
             Constants.QueryConstants.EVENT to Constants.QueryValues.EVENT_CLICK
-        ), encodedParams.toTypedArray()).subscribeOn(Schedulers.io())
+        ), encodedParams.toTypedArray(), resultID).subscribeOn(Schedulers.io())
```

Comment:
The existing system seems to be designed to pass optional parameters into an `encodedParams` array instead of adding extra params to the data manager calls